usaspending_api/disaster/v2/views/award/amount.py

```diff
@@ -22,17 +27,24 @@ def queryset(self):
             self.all_closed_defc_submissions,
             self.has_award_of_provided_type,
             self.is_in_provided_def_codes,
-            self.is_non_zero_award_spending,
         ]
 
-        count_field = self.unique_file_c
```

Comment:
![Alt Text](https://media.giphy.com/media/BIZkwFtu2xDlS/giphy.gif)

Reply:
This kept throwing me off. It's branching logic, it should look like it.

Reply:
I have faith you'll learn to read code one day